src/client.cpp

```diff
@@ -1010,11 +971,11 @@ void CClient::AudioCallback ( CVector<int16_t>& psData, void* arg )
     // process audio data
     pMyClientObj->ProcessSndCrdAudioData ( psData );
 
-/*
```

Comment:
Any test block shouldn't be indented.

Reply:
ok that's just a few will exclude them also